src/security/PeerConnector.cc

```diff
@@ -57,6 +57,8 @@ Security::PeerConnector::PeerConnector(const Comm::ConnectionPointer &aServerCon
     comm_add_close_handler(serverConn->fd, closeHandler);
 }
 
+Security::PeerConnector::~PeerConnector() = default;
```

Comment:
> Please try to do this in the class definition so it can be inlined or optimized away when relevant.

@chtsanti, TlsNegotiationDetails is gone now, but please check whether the _essence_ of the explanation of the current destruction definition location in branch commit fb8d208 no longer applies:

>  Placing defaulted destructor in the .cc file is necessary to avoid -Wdelete-incomplete errors that the default placement in the header file would trigger. To generate the destructor, the compiler needs to know full TlsNegotiationDetails declaration, and that knowledge is available in .cc file only.

This job destructor is very unlikely to be ever inlined because AsyncJobs are destroyed by job-management code that does not know the exact job type. We should not inline its definition if inlining it requires adding more `#include` statements to PeerConnector.h. And vice versa: We should inline if it does not hurt.

Reply:
Requires the <Security/Io.h> include.
I did not do this change for for now.

Reply:
Okay then. Thank you.